packages/web/.env/.env.mobile.prod

```diff
@@ -1,4 +1 @@
-REACT_APP_NATIVE_MOBILE=true
-REACT_APP_NATIVE_NAVIGATION_ENABLED=true
-
-PORT=3012
\ No newline at end of file
+PORT=3012
```

Comment:
Can probably remove this and the corresponding command entirely

Reply:
oh yeah totes!!!